src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicWinForms.cs

```diff
@@ -14,7 +14,12 @@ public class BasicWinForms : AbstractEditorTest
         protected override string LanguageName => LanguageNames.VisualBasic;
 
         public BasicWinForms(VisualStudioInstanceFactory instanceFactory)
-            : base(instanceFactory, nameof(BasicWinForms), WellKnownProjectTemplates.WinFormsApplication, clearEditor: false)
+            : base(
+                  instanceFactory,
+                  nameof(BasicWinForms),
+                  WellKnownProjectTemplates.WinFormsApplication,
+                  clearEditor: false,
+                  setUseSuggestionMode: false)
```

Comment:
This feels like we're working around a flakiness issue we don't understand. The fact this should be reset does or doesn't impact the test.